Extension/lib/settings/sync-service.js

```diff
@@ -534,11 +561,7 @@
         if (providerService.oauthSupported) {
             if ((!token || !api.oauthService.setToken(providerName, token, securityToken, expires))
                 && !api.oauthService.isAuthorized(providerName)) {
-                adguard.tabs.create({
-                    active: true,
-                    type: 'popup',
-                    url: api.oauthService.getAuthUrl(providerName, 'http://testsync.adguard.com/oauth?provider=' + providerName)
-                });
+                openAuthTab(providerName);
```

Comment:
We should not call `openAuthTab` here because this method is invoked on startup of the extension.